### PR TITLE
MULE-13575: File connector tests using locks fail on Windows

### PR DESCRIPTION
**Pull request by marianogonzalez**

### CODE REVIEW

File: src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java [hunk 1]
```diff
@@ -55,52 +57,54 @@ public void write(String filePath, InputStream content, FileWriteMode mode,
...
+                                                 path),
+                                          e);
     } catch (FileSystemException e) {
       //The only way to be sure that the exception was raised due to an illegal path.
       if (IS_A_DIRECTORY_MESSAGE.equals(e.getReason())) {
         throw new IllegalPathException(format("Cannot write to path '%s' because it is a Directory.", path), e);
       }
 
-      throw e;
+      throw exception(format("Exception was found writing to file '%s'", path), e);
```
Comment (alegmarra) [inline src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java]: file write error?

File: src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java [hunk 2]
```diff
@@ -55,52 +57,54 @@ public void write(String filePath, InputStream content, FileWriteMode mode,
...
+    } catch (Exception e) {
+      throw exception(format("Exception was found writing to file '%s'", path), e);
```
Comment (alegmarra) [inline src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java]: file write error?

File: src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java (hunk 1 above)
Comment (marianogonzalez) [reply src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java]: not

File: src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java (hunk 2 above)
Comment (marianogonzalez) [reply src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java]: not

**Pull request merged**